### PR TITLE
Added Stablebase for Base

### PR DESCRIPTION
**Pull request by PatelPrinci**
## Thank you for contributing to Spellbook 🪄
Please open the PR in **draft** and mark as ready when you want to request a review. 

### Description:

[...]

---
quick links for more information:
- [README.md](https://github.com/duneanalytics/spellbook/blob/main/README.md)
- [spellbook docs](https://github.com/duneanalytics/spellbook/tree/main/docs)
- [CONTRIBUTING.md](https://github.com/duneanalytics/spellbook/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by jeff-dude** [comment]
hi @PatelPrinci welcome to spellbook!

please comment the below in a standalone PR comment in order to sign the license agreement:
`I have read the CLA Document and I hereby sign the CLA`

File: dbt_subprojects/dex/models/trades/base/platforms/stablebase_base_base_trades.sql [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+WITH token_swaps AS (
+    SELECT
+        evt_block_number AS block_number,
+        evt_block_time AS block_time,
+        evt_tx_from AS maker,
+        evt_tx_to AS taker,
+        tokensSold AS token_sold_amount_raw,
+        tokensBought AS token_bought_amount_raw,
+        soldId AS token_sold_id,
+        boughtId AS token_bought_id,
```
Comment (jeff-dude) [inline dbt_subprojects/dex/models/trades/base/platforms/stablebase_base_base_trades.sql:20]:
```suggestion
        soldId AS token_sold_address,
        boughtId AS token_bought_address,
```

File: dbt_subprojects/dex/models/trades/base/platforms/stablebase_base_base_trades.sql [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    'stablebase' AS project,
+    '1' AS version,
+    CAST(date_trunc('month', token_swaps.block_time) AS date) AS block_month,
+    CAST(date_trunc('day', token_swaps.block_time) AS date) AS block_date,
+    token_swaps.block_time,
+    token_swaps.block_number,
+    token_swaps.token_sold_amount_raw,
+    token_swaps.token_bought_amount_raw,
+    token_swaps.token_sold_id,
+    token_swaps.token_bought_id,
```
Comment (jeff-dude) [inline dbt_subprojects/dex/models/trades/base/platforms/stablebase_base_base_trades.sql:43]:
```suggestion
    token_swaps.token_sold_address,
    token_swaps.token_bought_address,
```

File: dbt_subprojects/dex/seeds/trades/stablebase_base_base_trades_seed.csv [hunk 3]
```diff
@@ -0,0 +1,4 @@
+blockchain,project,version,block_month,block_date,block_time,block_number,token_sold_amount_raw,token_bought_amount_raw,token_sold_id,token_bought_id,maker,taker,project_contract_address,tx_hash,evt_index
```
Comment (jeff-dude) [inline dbt_subprojects/dex/seeds/trades/stablebase_base_base_trades_seed.csv:1]:
```suggestion
blockchain,project,version,block_month,block_date,block_time,block_number,token_sold_amount_raw,token_bought_amount_raw,token_sold_address,token_bought_address,maker,taker,project_contract_address,tx_hash,evt_index
```

File: dbt_subprojects/dex/models/trades/base/platforms/stablebase_base_base_trades.sql [hunk 4]
```diff
@@ -0,0 +1,50 @@
+{{ config(
+    schema = 'stablebase',
```
Comment (jeff-dude) [inline dbt_subprojects/dex/models/trades/base/platforms/stablebase_base_base_trades.sql:2]:
```suggestion
    schema = 'stablebase_base',
```

File: dbt_subprojects/dex/models/trades/base/platforms/stablebase_base_base_trades.sql [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    alias = 'token_swap_events',
```
Comment (jeff-dude) [inline dbt_subprojects/dex/models/trades/base/platforms/stablebase_base_base_trades.sql:3]:
```suggestion
    alias = 'base_trades',
```

**Comment by PatelPrinci** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by jeff-dude — APPROVED** [review]

**Comment by jeff-dude** [comment]
i don't have access to your branch to resolve merge conflicts that came from a recent merge. plz do fix the conflicts and push to the PR, so i can help merge 🙏

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**